### PR TITLE
F #2808: add DD_BLOCK_SIZE option

### PR DESCRIPTION
**Pull request by kvaps**
Add `DD_BLOCK_SIZE` option for allow specifying block size for the dd commands.

Fixes: https://github.com/OpenNebula/one/issues/2808

### CODE REVIEW

File: src/datastore_mad/remotes/fs/fs.conf [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# -------------------------------------------------------------------------- #
```
Comment (rsmontero) [inline src/datastore_mad/remotes/fs/fs.conf]: Is this file included in install.sh?

File: src/market_mad/remotes/http/import [hunk 2]
```diff
@@ -77,7 +77,7 @@ if [ -n "$BRIDGE_LIST" ]; then
     DST_HOST=`get_destination_host $ID`
     CP_CMD="$UTILS_PATH/downloader.sh -n '${IMPORT_SOURCE}' -"
 
-    multiline_exec_and_log "set -e -o pipefail; $CP_CMD | $SSH ${DST_HOST} $DD of=${DST_PATH} bs=64k" \
+    multiline_exec_and_log "set -e -o pipefail; $CP_CMD | $SSH ${DST_HOST} $DD of=${DST_PATH} bs=${DD_BLOCK_SIZE:-64k}" \
```
Comment (rsmontero) [inline src/market_mad/remotes/http/import]: Where could this DD_BLOCK_SIZE set? Is there any conf file for market?

File: src/datastore_mad/remotes/fs/fs.conf (hunk 1 above)
Comment (kvaps) [reply src/datastore_mad/remotes/fs/fs.conf]: nope

File: src/market_mad/remotes/http/import (hunk 2 above)
Comment (kvaps) [reply src/market_mad/remotes/http/import]: > Where could this DD_BLOCK_SIZE set
for now the only way to set is via global environment variables

> Is there any conf file for market?
Still not, but I can prepare it.

**Pull request merged**